src/main/java/blob/Parser.java

```diff
@@ -0,0 +1,258 @@
+                    StringBuilder str = new StringBuilder(" " + arr[l]);
+                    a = a.append(str);
+                }
+
+                // for task deadline string
+                StringBuilder sDate = new StringBuilder(arr[by + 1]);
+                StringBuilder sTime = new StringBuilder(arr[by + 2]);
+                String ISO8601Format = sDate.toString() + "T" + sTime.toString() + ":00";
+
+                try {
+                    Deadline d = new Deadline(a.toString(), false, ISO8601Format);
+                    tasklist.addTask(d);
+                    msg.append("Got it. I've added this task:\n");
+                    msg.append(d + "\n");
+                    msg.append("Now you have " + tasklist.size() + " tasks in the list.");
+                } catch (DateTimeParseException e) {
+                    msg.append("Input dates and times not in the format 'yyyy-mm-dd HH:mm'!");
+                }
+                return msg.toString();
+
+            } else if (act.equals("event")) {
+                int start = 0;
+                int end = 0;
+                for (int j = 0; j < arr.length; j++) {
+                    if (arr[j].equals("/from")) {
+                        start = j;
+                    } else if (arr[j].equals("/to")) {
+                        end = j;
+                    }
+                }
+
+                // task name error handling
+                if (start <= 1) {
+                    return "What's the name of your event? Please add it 😅!";
+                }
+
+                // start string error handling
+                if (end - start <= 1) {
+                    return "When does your event begin? Please add it 😅!";
+                }
+
+                // end string error handling
+                if (end == arr.length - 1) {
+                    return "What time does your event end? Please let me know 😅!";
+                }
+
+                //'/from' only having date or time
+                if (end - start == 2) {
+                    return "You need to add both DATE and TIME to your event's start \uD83D\uDD27! (Format: yyyy-mm-dd HH:mm)";
+                }
+
+                //'/to' only having date or time
+                if (end == arr.length - 2) {
+                    return "You need to add both DATE and TIME to your event's end \uD83D\uDD27! (Format: yyyy-mm-dd HH:mm)";
+                }
+
+                StringBuilder msg = new StringBuilder("");
+
+                //for task name string
+                StringBuilder a = new StringBuilder(arr[i + 1]);
+                for (int l = i + 2; l < start; l++) {
+                    StringBuilder str = new StringBuilder(" " + arr[l]);
+                    a = a.append(str);
+                }
+
+                //for start string
+                StringBuilder stDate = new StringBuilder(arr[start + 1]);
+                StringBuilder stTime = new StringBuilder(arr[start + 2]);
```

Comment:
Maybe can make the name become more explainable to what it do, like stDate can become startDate, stTime can become startTime